src/splunk-events.ts

```diff
@@ -77,57 +101,88 @@ interface SplunkEvent {
   event: EventData | string
 }
 
+const DEFAULT_EXPONENTIAL_BACKOFF_LIMIT = 60_000
+const DEFAULT_DEBOUNCE_TIME = 2_000
```

Comment:
Could you add the order of magnitude? Given that this was the root cause of the bug, being explicit about it looks good to me.

If you agree, I'm ok with adding this as a suffix to the constant name or adding a commentary.

Reply:
I think it would be better to add this in the JSDoc for the config object, wdyt? This way, the users of this lib would know this even if they don't look in our source code

Reply:
Looks great!